### DISCUSSION TITLE
[Help]: how to set up the environment variable and map storage to the container?

### DISCUSSION THREAD

**Discussion by 14paavang** [mbentley/docker-omada-controller#428] [Help] [answered]
### Controller Version

Not installed yet

### Describe Your Issue or Question

What are the required environment variables to get it to work? Also can the ports inside the container be changed? This is specifically because I plan to run this I. Truenas scale and it doesn’t let you map ports that are lower than 9000. Also do I need to kap any storage directories to the container?

### Expected Behavior

Idk, I havent installed it yet 

### Steps to Reproduce

None yet

### How You're Launching the Container

```plain
I was just going to run the fetch command on the built in gui on truenas scale
```

### Container Logs

```plain
None yet
```

### Additional Context

I am a Linux and docker noob

**Comment by mbentley** [comment, marked as answer]
It’s all in the readme:
Ports https://github.com/mbentley/docker-omada-controller?tab=readme-ov-file#using-non-default-ports
Variables https://github.com/mbentley/docker-omada-controller?tab=readme-ov-file#optional-variables

Look at the example run commands and it mentions two persistent data paths that are shown as volumes. Either bind mount those directly to the filesystem or use docker named volumes.
